.golangci.yml

```diff
@@ -99,6 +99,18 @@ linters-settings:
         deny:
           - pkg: "github.com/kumahq/kuma/app"
             desc: "pkg can't import app"
+      DenyGolangProtobuf:
+        files:
+          - $all
+        deny:
```

Comment:
Not a specialist here but is it possible to add 
```yaml
allow:
  - "github.com/golang/protobuf/jsonpb"
```

and avoid `// nolint: depguard` ?

Reply:
However, I think it's also a note for the user to know what happend for the `json` sdk and be more careful. Does that make sense?

PS: Get a try locally but it doesn't work well (It seems like a bug for golanglint and it gave lots of other unreleated packages warnings)